File: cpp_solutions/2037.cpp
class Solution {
public:
    int minMovesToSeat(vector<int>& seats, vector<int>& students) {
        sort(seats.begin(), seats.end());
        sort(students.begin(), students.end());
        int n = seats.size();
        int res = 0;
        for (int i = 0; i < n; ++i) {
            res += abs(seats[i] - students[i]);
        }
        return res;
    }
};

File: rust_solutions/2037.rs
impl Solution {
    pub fn min_moves_to_seat(mut seats: Vec<i32>, mut students: Vec<i32>) -> i32 {
        seats.sort_unstable();
        students.sort_unstable();
        seats.iter().zip(students.iter()).map(|(seat, student)| (seat - student).abs()).sum()
    }
}
